tests/tracer.go

```diff
@@ -36,13 +38,22 @@ func tcpEventCbV6(e tracer.TcpV6) {
 }
 
 func main() {
-	if len(os.Args) != 2 {
-		fmt.Fprintf(os.Stderr, "Usage: %s .../tcptracer-ebpf.o\n", os.Args[0])
+	if len(os.Args) != 1 {
+		fmt.Fprintf(os.Stderr, "Usage: %s\n", os.Args[0])
 		os.Exit(1)
 	}
-	fileName := os.Args[1]
 
-	t, err := tracer.NewTracerFromFile(fileName, tcpEventCbV4, tcpEventCbV6)
+	asset, err := tracer.TracerAsset()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "%v\n", err)
+		os.Exit(1)
+	}
+	hash := sha512.New()
```

Comment:
I don't think this is too useful (particularly in the example code), I was thinking about the git hash, but since it's harity to get I'd rather do nothing.